lms/templates/admin/macros.html.jinja2

```diff
@@ -118,8 +118,8 @@
 {% macro instances_table(request,  instances) %}
 {{ object_list_table(
     request, 'admin.instance', instances,
-    ['Name', 'Consumer key', 'GUID', 'Deployment ID'],
-    ['name', 'consumer_key', 'tool_consumer_instance_guid', 'deployment_id']
+    ['Name', 'Consumer key', 'GUID', 'Deployment ID', 'Last Launched'],
+    ['name', 'consumer_key', 'tool_consumer_instance_guid', 'deployment_id', 'last_launched']
```

Comment:
I'd say that in the context of listing multiple AI the least differentiating attribute would be the GUID.